dashboard/app/controllers/safe_browsing_controller.rb

```diff
@@ -0,0 +1,10 @@
+class SafeBrowsingController < ApplicationController
+  include SafeBrowsing
+  skip_before_action :verify_authenticity_token
```

Comment:
This worries me a little. Do we have any other protections in place to prevent, anyone, anywhere on the internet for going through us to use the google safe browsing API without an API key?

Reply:
Turns out, I could take that out that skip action and things would still work. :+1: